.github/workflows/performance-tests.yml

```
@@ -0,0 +1,530 @@
+
+  # Performance testing job
+  performance-tests:
+    name: Performance Tests
+    runs-on: ubuntu-latest
+    needs: [setup, wait-for-deployment]
+    if: always() && needs.setup.result == 'success' && (needs.wait-for-deployment.result == 'success' || needs.wait-for-deployment.result == 'skipped')
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        test-group: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)